dexs/saucerswap/index.ts

```diff
@@ -1,45 +1,57 @@
 import { ChainBlocks, FetchOptions, SimpleAdapter } from "../../adapters/types";
 import { CHAIN } from "../../helpers/chains";
-import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
-import { getPrices } from "../../utils/prices";
 import { httpGet } from "../../utils/fetchURL";
 
-const historicalVolumeEndpoint = (to: number) =>`https://server.saucerswap.finance/api/public/stats/platformData?field=VOLUME&interval=DAY&from=1650586&to=${to}`
-// https://server.saucerswap.finance/api/public/stats/platformData?field=VOLUME&interval=DAY&from=1650586&to=1682093355
-interface IVolumeall {
+const methodology = {
+  Fees: "A 0.3% fee is charged to users on every swap, 5/6 goes to liquidity providers, 1/6 goes to protocol.",
+}
+
+const historicalVolumeEndpoint = (to: number) => `https://server.saucerswap.finance/api/public/pools/platform-data?field=VOLUME_USD&interval=DAY&from=1650586&to=${to}`
```

Comment:
passing a custom from parameter here reduces the records requested? smol optimization